### ISSUE TITLE
No argument methods - cursor position

### ISSUE THREAD

**Issue by Jurigag** [klesun/deep-assoc-completion#107]
When method is resolved by deep assoc plugin and have no arguments it puts cursor anyway in method call, so if you want to do chain call you end with something like:

![image](https://user-images.githubusercontent.com/10086991/62128818-8aebd400-b2d5-11e9-9c7a-c1682a5e43d1.png)

**Comment by klesun** [comment]
Thanks for your feedback. Will fix that in next release.

**Comment by klesun** [comment]
Minimal test case to reproduce the issue:
```php
<?php

class CampaignGroupId
{
    public function getId()
    {
        return 345;
    }
}

class CampaignGroup
{
    public function getCampaignGroupId()
    {
        return new CampaignGroupId();
    }
}

class CampaignGroupQueryService
{
    public function getCampaignGroupsQueryByBrandMasterId($masterBrandId)
    {
        return [new CampaignGroup];
    }
}

class SomeCls
{
    /** @var CampaignGroupQueryService */
    private $campaignGroupQueryService;

    public function main()
    {
        $masterBrandId = 123;
        $campaignGroups = $this->campaignGroupQueryService->getCampaignGroupsQueryByBrandMasterId($masterBrandId);
        $data = [];

        foreach ($campaignGroups as $campaignGroup) {
            $data[] = $campaignGroup->getCampaignGroupId();
        }
    }
}
```

**Comment by klesun** [comment]
Fix released in [2019.07.31.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/66473)
